gateway/1.5/modules/ROOT/pages/flex-troubleshoot-logging.adoc

```diff
@@ -6,8 +6,32 @@ endif::[]
 
 When configuring logging for Anypoint Flex Gateway, you might encounter logging issues caused by incorrect directory permissions, and so on. The following topics provide possible causes for logging issues and describe how to troubleshoot:
 
+* <<third-party-service, Logs Do Not Appear in a Third-Party Logging Service>>
 * <<incorrect-permissions,Logs Do Not Appear when Running in a VM>>
 
+[[third-party-service]]
+== Logs Do Not Appear in a Third-Party Logging Service
+
+After applying a logging configuration YAML file, logs should begin appearing within a few minutes. If your logs do not appear in your third party logging service:
+
+. Ensure that your third-party service is running. How you do this is dependent on your chosen service.
```

Comment:
I'd reword it to something like:
"Refer to your chosen service documentation to learn more about this topic."